tests/_utils/send_lark_notify.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+)
+
+func main() {
+	url := os.Getenv("FEISHU_BOT_WEBHOOK_URL")
+	repo := os.Getenv("GITHUB_REPOSITORY")
+	committer := os.Getenv("GITHUB_ACTOR")
+	jobID := os.Getenv("GITHUB_RUN_ID")
+	jobURL := fmt.Sprintf("https://github.com/%s/actions/runs/%s", repo, jobID)
+	msgTMPL := `
+	{
+		"msg_type": "post",
+		"content": {
+		  "post": {
+			"zh_cn": {
+			  "title": "FIX ME",
+			  "content": [
+				[
+				  {
+					"tag": "text",
+					"text": "repo :"
+				  },
+				  {
+					"tag": "text",
+					"text": "%s"
+				  }
+				],
+				[
+				  {
+					"tag": "text",
+					"text": "committer :"
+				  },
+				  {
+					"tag": "text",
+					"text": "%s"
+				  }
+				],
+				[
+				  {
+					"tag": "text",
+					"text": "url :"
+				  },
+				  {
+					"tag": "a",
+					"text": "link",
+					"href": "%s"
+				  }
+				]
+			  ]
+			}
+		  }
+		}
+	  }`
+	jsonStr := fmt.Sprintf(msgTMPL, repo, committer, jobURL)
+	req, err := http.NewRequest("POST", url, bytes.NewBuffer([]byte(jsonStr)))
```

Comment:
when this `err` is not nil, can we discover the failure such as a nil `req`?

Reply:
This line of code will basically not fail, because all input params is checked before this line